templates/zerver/compose.html

```diff
@@ -95,12 +95,12 @@
                   <a class="message-control-button icon-vector-font" title="{{ _('Formatting') }}" data-overlay-trigger="markdown-help"></a>
```

Comment:
Not sure where to put this comment, but while you're at it, would be great to add an "Add emoji" title to the icon in the bottom left of the compose box